src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -31,6 +38,7 @@
     <Compile Include="System\Runtime\InteropServices\ExporterEventKind.cs" />
     <Compile Include="System\Runtime\InteropServices\RegistrationClassContext.cs" />
     <Compile Include="System\Runtime\InteropServices\RegistrationConnectionType.cs" />
+    <Compile Include="System\Runtime\InteropServices\StandardOleMarshalObject.cs" />
```

Comment:
I suppose these types aren't for uap scenario. Could you add a condition for these types?